client/src/components/Dashboard/Calendar.js

```diff
@@ -27,6 +27,15 @@ function classNames(...classes) {
 
 let colStartClasses = ['', 'col-start-2', 'col-start-3', 'col-start-4', 'col-start-5', 'col-start-6', 'col-start-7'];
 
+function selectedDate(date) {
```

Comment:
@yli0402 can you move this `selectedDate` function to `client\src\shared\services\Helpers.js`, and put the `monthList` array into a `Months.js` and put that file under `client\src\shared\constants`?

you can then reference that file in the function, and call that function in this file from `Helpers.js`

since this is a generic function and that the months are just constants, i think it'd make the code cleaner and also other developers can reference them in case they need to in the future

Reply:
finished!